examples/cloudflare-modules/src/index.ts

```diff
@@ -1,22 +1,27 @@
 // src/index.mjs
 import { createYoga, createSchema } from 'graphql-yoga'
 
-const { fetch } = createYoga({
-  graphqlEndpoint: '/graphql',
-  landingPage: false,
-  schema: createSchema({
-    typeDefs: /* GraphQL */ `
-      type Query {
-        greetings: String
-      }
-    `,
-    resolvers: {
-      Query: {
-        greetings: () =>
-          'This is the `greetings` field of the root `Query` type',
-      },
+const schema = createSchema({
+  typeDefs: /* GraphQL */ `
+    type Query {
+      greetings: String
+    }
+  `,
+  resolvers: {
+    Query: {
+      greetings: () => 'This is the `greetings` field of the root `Query` type',
     },
-  }),
+  },
 })
 
-export default { fetch }
+export default {
+  fetch(request: Request, env: { [key: string]: string }) {
+    const yoga = createYoga({
+      graphqlEndpoint: env.GRAPHQL_ROUTE || '/graphql',
```

Comment:
We do `landingPage: false` and then `GRAPHQL_ROUTE` is injected to tell the Yoga server to expect incoming connections on `e2e.graphql-yoga.com/WORKER_ID` 
This way we can have a public endpoint easily, without the need to capture `/WORKER_ID/*`. 
It's a workaround, but it works for now...